src/Arcus.Security.Secrets.AzureKeyVault/KeyVaultSecretProvider.cs

```diff
@@ -38,6 +43,10 @@ public KeyVaultSecretProvider(IKeyVaultAuthenticator authenticator, IKeyVaultCon
             VaultUri = $"{vaultConfiguration.VaultUri.Scheme}://{vaultConfiguration.VaultUri.Host}";
 
             _vaultConfiguration = vaultConfiguration;
+
+            /** TODO: if we only allow a single instance (KeyVaultClient) being created for this type,
```

Comment:
Because those hand the instantiation which also defines how it authenticates

Reply:
See my previous comment(s)

Reply:
Can you point out which one please?

Reply:
Merged in recent changes and this comment is no longer applicable I think